### ISSUE TITLE
Bundle breaks when reimporting renamed folder

### ISSUE THREAD

**Issue by fabrizzio-dotCMS** [dotCMS/core#21886]
**Describe the bug**
Reproducible in master and release-22.04

if you create a folder and add some file assets to it then generate a **bundle** and have it imported on the destination dotcms all good. Then rename the same folder on the source instance and regenerate the bundle and once again have it imported on the destination instance. An exception occurs and the Bundle fails.

**To Reproduce**
1. Take demo as the source and a local instance of master as the destination
2. On the Source Go to Site-Browser make a folder named **test** and add some file assets to it.
3. Make a bundle from the test folder. Now have it imported from the destination instance. So far so good.
4. Go back to the source instance and rename the folder test to something like **test-2** and regenerate a new bundle
5. From the destination instance have it imported once again. you'll see how the import fails with the following exception:
 

```
ESContentletAPIImpl: demo.dotcms.com : Saving Content , StartDate: N/D; EndDate: N/D; User:system; ContentIdentifier: 74d0e64a5b26ab62839553f5ce562207
19:40:35.698  ERROR business.ESContentletAPIImpl - ERROR: Cannot insert/update for this path does not exist for the given host
  Where: PL/pgSQL function identifier_parent_path_check() line 13 at RAISE{
  "SQL": ["UPDATE identifier set parent_path=?, asset_name=?, host_inode=?, asset_type=?, syspublish_date=?, sysexpire_date=?, owner=?, create_date=?, asset_subtype=? where id=?"],
  "maxRows": [-1],
  "offest": [0],
  "params": [
    "/test/",
    "bats-needs-a-ride.jpg",
    "48190c8c-42c4-46af-8d1a-0cd5db894797",
    "contentlet",
    null,
    null,
    "system",
    "2022-03-23 01:39:24.481",
    "FileAsset",
    "74d0e64a5b26ab62839553f5ce562207"
  ]
}
com.dotmarketing.exception.DotDataException: ERROR: Cannot insert/update for this path does not exist for the given host
  Where: PL/pgSQL function identifier_parent_path_check() line 13 at RAISE{
  "SQL": ["UPDATE identifier set parent_path=?, asset_name=?, host_inode=?, asset_type=?, syspublish_date=?, sysexpire_date=?, owner=?, create_date=?, asset_subtype=? where id=?"],
  "maxRows": [-1],
  "offest": [0],
  "params": [
    "/test/",
    "bats-needs-a-ride.jpg",
    "48190c8c-42c4-46af-8d1a-0cd5db894797",
    "contentlet",
    null,
    null,
    "system",
    "2022-03-23 01:39:24.481",
    "FileAsset",
    "74d0e64a5b26ab62839553f5ce562207"
  ]
}
	at com.dotmarketing.common.db.DotConnect.loadResult(DotConnect.java:273) ~[dotcms_master_5cd6108.jar:?]
```

**Expected behavior**
Once the folder gets renamed we should be able to reimport and have the new folder created.

**Additional context**
Somehow the new bundle still carries the old folder name on the file assets
creating the conflict. Because there's a folder already named like that containing that file on the destination. 
sounds like an old issue to me.

If I have pushed a bundle with a folder that has been already renamed on sender containing an asset that still exists on the receiver within the old (already pushed folder). That situation still breaks pp.

**Comment by nollymar** [comment]
**PRs:** 
https://github.com/dotCMS/core/pull/21912
https://github.com/dotCMS/enterprise/pull/862

**Comment by fabrizzio-dotCMS** [comment]
Tested against the latest release code. And it's looking good.

**Comment by bryanboza** [comment]
Fixed, tested on release-22.04 // Docker // FF
